relay-server/src/actors/processor.rs

```diff
@@ -1279,8 +1249,82 @@ impl EnvelopeProcessorService {
         Ok(())
     }
 
+    /// Previous / Old implementation of the `process_state` function, which is used for all
+    /// unknown [`ProcessingGroup`] variant.
+    ///
+    /// Note: this will be removed once we are confident in the new implementation and make sure
+    /// that all the groups properly covered.
+    fn process_state_pre(&self, state: &mut ProcessEnvelopeState) -> Result<(), ProcessingError> {
```

Comment:
nit: I would rename this to `process_state_old` or `process_state_legacy`. When I first saw the function name I thought it was a pre-processor.

Reply:
done